src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the `loading` state variable and the `profile` context variable seems appropriate. The `loading` state is used to control the loading states of different actions, and the `profile` context is used to access profile-related data. The hooks `useCreateChat` and `useDMMessage` are also correctly initialized.
```diff
+ const [loading, setLoading] = React.useState({
+   loadingDm: false,
+   loadingDmAnon: false
+ });
+ 
+ const [profile] = React.useContext(Context).profile;
+ const {createSignChat} = useCreateChat();
+ const {sendMessageDM} = useDMMessage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->